strax/processing/hitlets.py

```diff
@@ -620,7 +502,12 @@ def highest_density_region_width(data,
     """
     res = np.zeros((len(fractions_desired), 2), dtype=np.float32)
     data = np.maximum(data, 0)
-    inter, amps = strax.highest_density_region(data, fractions_desired, _buffer_size=_buffer_size)
+    
+    if np.all(data == 0):
+        res[:] = np.nan
+    else:
+        inter, amps = strax.highest_density_region(data,
+                                                   fractions_desired, _buffer_size=_buffer_size)
```

Comment:
Sorry, I know one should not add more than one feature into a PR, but I am currently optimizing some splitting parameters and I run into this issue. `strax.highest_density_region` behaves as expected and raises an ValueError. However, it is not desired to stop the entire processing because of that. Hence, the function returns now np.nan in case the HDR is not defined.